src/mixer/state.ts

```diff
@@ -454,7 +465,18 @@ export const load = (
     const blob = new Blob([rawData]);
     const objectUrl = URL.createObjectURL(blob);
 
-    const channelOutputId = getState().settings.channelOutputIds[player];
+    let channelOutputId = getState().settings.channelOutputIds[player];
+    // If the player setting doesn't exist, reset the settings. (Happens after player count is increased)
```

Comment:
At some point we'll need to write a proper settings "upgrade / migration" system (e.g. making saving the default). Out of scope of this PR though.

Reply:
Indeed, my assumption was that new variables would get auto filled, but that's not the case. I was hoping to just rename the variable and set the initial to true. Sounds like this won't be the case.